monitoring/uss_qualifier/scd/reports.py

```diff
@@ -57,28 +58,49 @@ class Interaction(ImplicitDict):
     interaction_id: InteractionID
     """ID of this interaction (used to refer to this interaction from an issue)"""
 
-    test_id: str
-    """ID of test for which this interaction was performed"""
+    context: AutomatedTestContext
+    """Context in which this interaction was performed"""
 
     test_step: int
-    """Step of test for which this interaction was performed"""
+    """Step of test for which this interaction was performed.
```

Comment:
In terms of readability of the reports, what do you think about either having an additional [`test_step_name`](https://github.com/interuss/dss/blob/dd807cc51dd85e00809ccdae656304aab9bc1faa/monitoring/uss_qualifier/scd/data_interfaces.py#L66) field, or else replacing this index with that name?  I'm not particularly fond of duplicating information in general, but the `context` does have both `test_id` and `test_name` which are technically redundant (`test_name` can be determined from `test_id`), but I think that still works well so it seems reasonable to also have the technically-redundant index and name both in the report.

Reply:
test_step_name added.